src/index.js

```diff
@@ -24,12 +26,27 @@ const parseChangelog = util.promisify(changelogParser);
  */
 const getBody = async (changelogPath, notesPath) => {
     if (Boolean(notesPath)) {
+        core.info(`Notes Path found: ${notesPath}`);
+
         try {
-            const releaseNotes = await fse.readFile(notesPath, 'utf-8');
+            const outputContent = [];
+            const filePaths = await globPromise(`${notesPath}*.md`);
+
+            core.debug(filePaths);
```

Comment:
do we need keep debugs?

Reply:
@gotbahn please yes it would be easier to catch it next time if we can run debug without changing the code